### PR TITLE
feat(webapp): 🍰 Search Users To Add To Group

### LINKED ISSUE #5629: 🔧 [Refactor] Reuse User Search For Group Add Member
## 🔧 Refactor ticket

Reuse user search for group add member so that you do not need to correct user slug but can use parts of the slug to find the member

### PR DESCRIPTION
**Pull request by Mogge**
### Issues
- fixes #5629 

### Todo

- [X] None

### CODE REVIEW

**Comment by ogerly** [comment]
TODO: 

- the id of the selected user is written into the input field after clicking. 
- when searching again, the user is still in the selection. 

https://user-images.githubusercontent.com/1324583/201053039-a30b15c3-35c5-4555-a9bd-508cc7189267.mp4

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey, cool !!! @Mogge @ogerly 

I have some questions and some suggestions …

If I add a user who is already a group member their ID appears in the search field:

Bildschirmfoto 2022-11-23 um 14 50 27

File: webapp/locales/de.json [hunk 1]
```diff
@@ -451,6 +452,12 @@
     "membersCount": "Mitglied ::: Mitglieder",
     "membersListTitle": "Gruppenmitglieder",
     "membersListTitleNotAllowedSeeingGroupMembers": "Gruppenmitglieder unsichtbar",
+    "modal": {
+      "cancel": "Abbrechen",
+      "confirm": "Bestätigen",
```
[the remark this answers is not in the document]
Comment (Mogge) [reply webapp/locales/de.json:457]: No. May be we want to change the title of the modal. I am not happy with the title but had no better idea so far.

File: webapp/locales/en.json [hunk 2]
```diff
@@ -423,7 +425,7 @@
       "title": "Edit My Group"
     },
     "errors": {
-      "userAlreadyMember": "User @{slug} is already a member"
+      "userAlreadyMember": "User “{name}” is already a member"
```
Comment (Tirokk) [inline webapp/locales/en.json:428]:
```suggestion
      "userAlreadyMember": "User “{name}” is already a member!"
```

File: webapp/locales/de.json [hunk 3]
```diff
@@ -423,7 +425,7 @@
       "title": "Meine Gruppe ändern"
     },
     "errors": {
-      "userAlreadyMember": "Nutzer @{slug} ist bereits Mitglied"
+      "userAlreadyMember": "Nutzer „{name}“ ist bereits Mitglied"
```
Comment (Tirokk) [inline webapp/locales/de.json:428]:
```suggestion
      "userAlreadyMember": "Benutzer „{name}“ ist bereits Mitglied!"
```

**Review by Tirokk — CHANGES_REQUESTED** [review]
In general it looks really good! @Mogge 

The user ID still appears if you add a user who is already member.
Did you mean you have healed this or must it be that way?

Bildschirmfoto 2022-11-24 um 08 34 51

**Review by Tirokk — APPROVED** [review]
Than I approve dear and we have to fix the ID in the search field later.
Please make an issue for it.

🚀🚀🌟🌟

**Pull request merged**